apps/test/unit/templates/convertXmlToBlockly.js

```diff
@@ -21,4 +21,15 @@ describe('convertXmlToBlockly', function() {
     expect(container.innerHTML).to.not.equal(content);
     expect(container.getElementsByTagName('svg').length).to.equal(2);
   });
+
+  it('ignores comments in xml', function() {
+    // For example, those generated by React
+    const container = document.createElement('div');
+    const content =
+      '<xml><!-- react-text: 1 --><block type="variables_get"><!-- react-text: 2 --><title name="VAR"><!-- react-text: 3 -->i</title></block></xml>';
+    container.innerHTML = content;
+    convertXmlToBlockly(container);
+    expect(container.innerHTML).to.not.equal(content);
```

Comment:
Should there be an `expect(container.innerHTML).to.equal(content)` check before the `convertXmlToBlockly` call, just to prove that setting and getting innerHTML doesn't itself cause some transform (like stripping comment nodes)?